### ISSUE TITLE
Page list is empty (ReadComicOnline Extension)

### ISSUE THREAD

**Issue by nandito104** [inorichi/tachiyomi-extensions#97]
For some reason there are some comics that I can neither read nor download since I keep getting the "Page List is Empty" message

Currently Running:
Tachiyomi                                    version 0.6.1
Tachyomi: ReadComicOnline      version 1.0.1

**Comment by ddmgy** [comment]
Which comics are giving this error? I'm going through the list of most popular comics on ReadComicOnline, but I've yet to hit one that does not work.

**Comment by shadowmar** [comment]
I'm having the same error with all comics I tried reading on ReadComicOnline. Went through all the popular comics and none of them worked. It was working yesterday in the morning but last night I started getting the error. I thought it might've been a captcha issue like I had once before but I didn't have a captcha popup when I went to the site like last time.

Tachiyomi 0.6.1 (Stable & Dev versions)
ReadComicOnline 1.0.1

**Comment by nandito104** [comment]
I'm currently getting this error with the main series of Fables. As to other series I don't know since Fables is the only one I'm currently reading.

**Comment by ddmgy** [comment]
Perhaps I'm not having any issues because I did not previously have the extension installed.

Are you able to read anything if you go directly to the website on your device, @nandito104? Have you tried clearing the chapter cache? (Settings > Advanced > Clear chapter cache) Beyond that, I'm not really sure what else would fix the problem.

**Comment by ddmgy** [comment]
Hmm. I'm not really sure what else to do here. I'm still unable to reproduce the issue. Perhaps @inorichi or @null-dev would know more.

There are a number of old issues from the main tachiyomi repo that seem to represent similar problems, mainly having to do with CloudFlare. [This issue (#572)](https://github.com/inorichi/tachiyomi/issues/572) and a [follow-up (#696)](https://github.com/inorichi/tachiyomi/issues/696) have some more information that may be useful.

**Comment by voythas** [comment]
@ddmgy can you try downloading 'Shade, the Changing Girl' comic? I can read it on the site without any problems, but I can't download it through extension.

**Comment by ddmgy** [comment]
Yes, I had no problem downloading that comic.

**Comment by voythas** [comment]
I cannot download it at all, that is weird. Can I provide you logs somehow for further investigation?

**Comment by ddmgy** [comment]
If you have a way to capture logs, go for it. It might provide some helpful information, but I'm pretty sure that this is a CloudFlare issue, which is not something I know how to fix. There are some steps suggested in the two issues I linked above that may help, but I can't guarantee they will help here.

If there were a problem in the extension itself, I could fix it. The issue is that CloudFlare seems to have flagged your IP, and the only way to fix that is to go the site on your affected device and pass a captcha to prove you are a human. Even after passing the captcha, you may still have issues with the extension until you restart your router, or you'll have to wait a few days until the problem fixes itself on the site's side.

**Comment by j2ghz** [comment]
(I think @voythas is asking how to provide logs)
The only way I know of is to debug the app in Android Studio and look at the html it receives, which is not easy for non-devs. I guess you could also capture the traffic externally, but that isn't easy either.

**Comment by voythas** [comment]
@ddmgy My PC and my phone are connected to the same wifi, so IP is exactly the same. I even tried via mobile network, just to be sure (so it's different). I have no problems with watching the same comic via browser using the same IP, so I don't think it's IP based problem. I don't have to go through the captcha as well (neither on desktop browser nor mobile browser). Yet still, I can't download almost any of the comics.

I will ask my colleagues for help to provide logs, I happen to work in mobile software house.

**Comment by null-dev** [comment]
Steps for taking logs:

1. Open Tachiyomi and reproduce the error (do whatever you did to cause the error).
2. **Immediately** after the error occurs, go to the `Settings` app on your phone, tap on `Developer options` and then tap on `Take bug report`.
3. If asked about the type of bug report, choose `Interactive report`.
4. Open the notification shade and wait for the bug report to complete. Once finished, tap on the notification to send it.

**Comment by shadowmar** [comment]
I think it might be an IP issue that for some reason is only affecting Tachiyomi. 

While at a friend's house I noticed I could load ReadComicOnline content again in Tachiyomi on my tablet, so I tried again when I got home but it didn't work (the actual website continues to work fine in Firefox on my tablet, no Captchas either). So I used a VPN and tried loading some comics in Tachiyomi again on my home network and it started loading again without any issues at all.

**Comment by voythas** [comment]
@null-dev thank you for that guide. Here are my logs:
[bugreport-NJH47D-2017-08-27-19-11-40.zip](https://github.com/inorichi/tachiyomi-extensions/files/1254572/bugreport-NJH47D-2017-08-27-19-11-40.zip)

**Comment by null-dev** [comment]
**TLDR; I don't think the bug report includes the necessary info :(.**

Yeah, Tachiyomi won't log the HTTP page in this case. We will probably need to debug the source/write a custom version of the source that has logging built in.

For other devs who are curious (but I don't think you will find anything):
Stack trace:
```
08-27 19:10:32.615  1225 14049 I ActivityManager: START u0 {cmp=eu.kanade.tachiyomi.debug/eu.kanade.tachiyomi.ui.reader.ReaderActivity} from uid 10289 on display 0
08-27 19:10:32.929  1225  1331 I ActivityManager: Displayed eu.kanade.tachiyomi.debug/eu.kanade.tachiyomi.ui.reader.ReaderActivity: +275ms
08-27 19:10:32.944 18837 18863 D OpenGLRenderer: endAllActiveAnimators on 0x7f73e1d400 (RippleDrawable) with handle 0x7f745bb360
08-27 19:10:33.893 18837 18837 E ReaderActivity: java.lang.Exception: Page list is empty
08-27 19:10:33.893 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.ui.reader.ChapterLoader$loadChapter$2.call(ChapterLoader.kt:66)
08-27 19:10:33.893 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.ui.reader.ChapterLoader$loadChapter$2.call(ChapterLoader.kt:18)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.util.ActionObserver.onNext(ActionObserver.java:39)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:96)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:101)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4$1.onNext(OperatorOnErrorResumeNextViaFunction.java:118)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:77)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:101)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1$requestArbiter$1.request(OkHttpExtensions.kt:25)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.producers.ProducerArbiter.setProducer(ProducerArbiter.java:126)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4$1.setProducer(OperatorOnErrorResumeNextViaFunction.java:130)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:102)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at rx.Subscriber.setProducer(Subscriber.java:205)
08-27 19:10:33.894 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:45)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:13)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4.onError(OperatorOnErrorResumeNextViaFunction.java:142)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.exceptions.Exceptions.throwOrReport(Exceptions.java:212)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:50)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:33)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
08-27 19:10:33.895 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.896 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:230)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
08-27 19:10:33.897 18837 18837 E ReaderActivity: 	at java.lang.Thread.run(Thread.java:761)
08-27 19:10:33.898 18837 18837 E ReaderActivity: Caused by: rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: java.util.ArrayList.class
08-27 19:10:33.898 18837 18837 E ReaderActivity: 	at rx.exceptions.Exceptions.throwOrReport(Exceptions.java:188)
08-27 19:10:33.898 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:98)
08-27 19:10:33.898 18837 18837 E ReaderActivity: 	... 51 more
```

Another one:
```
08-27 19:11:23.434  1225  2559 I ActivityManager: START u0 {cmp=eu.kanade.tachiyomi.debug/eu.kanade.tachiyomi.ui.reader.ReaderActivity} from uid 10289 on display 0
08-27 19:11:23.644 18837 18837 E ReaderActivity: java.lang.Exception: Page list is empty
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.ui.reader.ChapterLoader$loadChapter$2.call(ChapterLoader.kt:66)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at eu.kanade.tachiyomi.ui.reader.ChapterLoader$loadChapter$2.call(ChapterLoader.kt:18)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.util.ActionObserver.onNext(ActionObserver.java:39)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:96)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:101)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.observers.Subscribers$5.onNext(Subscribers.java:235)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4.onNext(OperatorOnErrorResumeNextViaFunction.java:154)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.producers.SingleDelayedProducer.emit(SingleDelayedProducer.java:102)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.producers.SingleDelayedProducer.setValue(SingleDelayedProducer.java:85)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:48)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:33)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.Observable.unsafeSubscribe(Observable.java:10256)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:230)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at java.lang.Thread.run(Thread.java:761)
08-27 19:11:23.644 18837 18837 E ReaderActivity: Caused by: rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: java.util.ArrayList.class
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.exceptions.Exceptions.throwOrReport(Exceptions.java:188)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:98)
08-27 19:11:23.644 18837 18837 E ReaderActivity: 	... 35 more
```

**Comment by voythas** [comment]
Okay, so I made some more tests. It seems that you guy are right, it's IP based somehow. I tried downloading via VPN and it allowed me to download few more comics. After that it started giving me 'Page list is empty' error again.
Is there a limit of how many I can download in short time? Maybe that's the issue. IPs are getting banned or something for downloading too much in too short time?

Is it something that could be tested by you?

**Comment by j2ghz** [comment]
I just tried downloading 25 chapters each 60 pages and no problem. It's probably not a ban, but their CAPTCHA or cloudflare's. Even if we were able to deduce the limits, it wouldn't help, because they often change. It's in the interest of the website owner to make it not work with Tachiyomi, so that's what they do. You can try https://github.com/inorichi/tachiyomi/issues/958#issuecomment-325124830.

**Comment by voythas** [comment]
I tried that, it didn't ask me for captcha or anything (I could read the comics via web), the problem still persist.

**Comment by j2ghz** [comment]
The problem is that it's possible to recognize browser from tachiyomi. So you can get CAPTCHA on one but not the other. If that is the case there's little to do.

**Comment by replis** [comment]
i have the same error going on with this readcomiconline.to is giving me "page list is empty" error again. If there is any solution please help.

**Comment by j2ghz** [comment]
If you get page list empty check that you are on latest stable version of the app, latest version of the extension source, try opening the site in browser (see if there's captcha). If the problem can be solved by switching internet connections from e.g. WiFi to data or to a WiFi somewhere else then it's a captcha and nothing can be currently done about that

**Comment by replis** [comment]
I have the latest dev tachiyomi.

I get it worked. But before opening the site with browser you need to clear all cache and cookies from the beginning from your chrome browser and then open the comic in browser from your tachiyomi app.

Like you said, if you open it from mobile data you can use it too.
So its a Captcha problem.

**Comment by CarlosEsco** [comment]
closing as this seems resolved
